### PR TITLE
Multiplying-neurons

### PR DESCRIPTION
**Pull request by mtewes**

### CODE REVIEW

**Comment by mtewes** [comment]
@kuntzer , you want to have a look ?

**Comment by kuntzer** [comment]
Yep, I'll go through the files in a bit.

File: demo/testlayer_mult.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#print layer.report()
+
+#layer.zero()
+#print layer.report()
+
+# First neuron: simple product
+layer.weights[0, 0] = 1
+layer.weights[0, 1] = 1
+
+# Second neuron: product of almost-squres
```
Comment (kuntzer) [inline demo/testlayer_mult.py]: typo in comment almost-squ*a*res ;)

File: tenbilac/layer.py [hunk 2]
```diff
@@ -16,49 +16,104 @@
...
 		self.weights *= 0.0
 		self.biases *= 0.0
 	
+
+	def setidentity(self, onlyn=None):
+		"""
+		Sets the weights and biases so that this layer transports all its first nn inputs to the nn outputs.
+		This is meant to give a promising initial condition for training a calibration-like task. 
+		
+		:param onlyn: If set, limits the number of neurons to be set. By default, all are set.
```
Comment (kuntzer) [inline tenbilac/layer.py]: Why are you setting only the first n instead of passing which weights should be set to 1?

**Comment by mtewes** [comment]
OK, fixed the typo :) Let me know when this can be merged in. The issue #14 also gives a description of some of the new options, + we can skype, of course (although I'm just leaving for a talk in 7 minutes)

File: tenbilac/layer.py (hunk 2 above)
Comment (kuntzer) [inline tenbilac/layer.py]: Other than this, it's fine by me

**Pull request merged**